src/deneb/beacon_block.rs

```diff
@@ -0,0 +1,127 @@
+use crate::altair::SyncAggregate;
+use crate::capella::SignedBlsToExecutionChange;
+use crate::deneb::ExecutionPayload;
+use crate::kzg::KzgCommitment;
+use crate::phase0::{
+    Attestation, AttesterSlashing, Deposit, Eth1Data, ProposerSlashing, SignedVoluntaryExit,
+};
+use crate::primitives::{BlsSignature, Bytes32, Root, Slot, ValidatorIndex};
+use ssz_rs::prelude::*;
+
+#[derive(Default, Debug, Clone, SimpleSerialize, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct BeaconBlockBody<
+    const MAX_PROPOSER_SLASHINGS: usize,
+    const MAX_VALIDATORS_PER_COMMITTEE: usize,
+    const MAX_ATTESTER_SLASHINGS: usize,
+    const MAX_ATTESTATIONS: usize,
+    const MAX_DEPOSITS: usize,
+    const MAX_VOLUNTARY_EXITS: usize,
+    const SYNC_COMMITTEE_SIZE: usize,
+    const BYTES_PER_LOGS_BLOOM: usize,
+    const MAX_EXTRA_DATA_BYTES: usize,
+    const MAX_BYTES_PER_TRANSACTION: usize,
+    const MAX_TRANSACTIONS_PER_PAYLOAD: usize,
+    const MAX_WITHDRAWALS_PER_PAYLOAD: usize,
+    const MAX_BLS_TO_EXECUTION_CHANGES: usize,
+    const MAX_BLOBS_PER_BLOCK: usize,
+> {
+    pub randao_reveal: BlsSignature,
+    pub eth1_data: Eth1Data,
+    pub graffiti: Bytes32,
+    pub proposer_slashings: List<ProposerSlashing, MAX_PROPOSER_SLASHINGS>,
+    pub attester_slashings:
+        List<AttesterSlashing<MAX_VALIDATORS_PER_COMMITTEE>, MAX_ATTESTER_SLASHINGS>,
+    pub attestations: List<Attestation<MAX_VALIDATORS_PER_COMMITTEE>, MAX_ATTESTATIONS>,
+    pub deposits: List<Deposit, MAX_DEPOSITS>,
+    pub voluntary_exits: List<SignedVoluntaryExit, MAX_VOLUNTARY_EXITS>,
+    pub sync_aggregate: SyncAggregate<SYNC_COMMITTEE_SIZE>,
+    pub execution_payload: ExecutionPayload<
+        BYTES_PER_LOGS_BLOOM,
+        MAX_EXTRA_DATA_BYTES,
+        MAX_BYTES_PER_TRANSACTION,
+        MAX_TRANSACTIONS_PER_PAYLOAD,
+        MAX_WITHDRAWALS_PER_PAYLOAD,
+    >,
+    pub bls_to_execution_changes: List<SignedBlsToExecutionChange, MAX_BLS_TO_EXECUTION_CHANGES>,
+    pub blob_kzg_commitments: List<KzgCommitment, MAX_BLOBS_PER_BLOCK>,
```

Comment:
May have to update `MAX_BLOBS_PER_BLOCK` to `MAX_BLOB_COMMITMENTS_PER_BLOCK`. 
https://github.com/ethereum/consensus-specs/pull/3338/files